src/components/Label/Label.tsx

```diff
@@ -41,6 +41,9 @@ class Label extends UIComponent<any, any> {
     /** An icon label can format an Icon to appear before or after the text in the label */
     iconPosition: PropTypes.oneOf(['start', 'end']),
 
+    /** Label can have an icon. */
+    image: customPropTypes.some([PropTypes.string, PropTypes.object]),
```

Comment:
`customPropTypes.shorthandContent`, since it can support string, number, object, element, and is disallowed with children.